Extensions/easy_video_links.js

```diff
@@ -0,0 +1,69 @@
+
+	run: function() {
+		
+		this.running = true;
+		XKit.tools.init_css("easy_video_links");
+		XKit.interface.create_control_button("xkit-easy-video-links-button", this.button_icon, "Copy Video Address", "");
+		XKit.extensions.easy_video_links.init();
+		XKit.post_listener.add("easy_video_links", XKit.extensions.easy_video_links.do);
+		XKit.extensions.easy_video_links.do();
+		
+	},
+
+	init: function() {
+
+		$(document).on("click", ".xkit-easy-video-links-button", function(event) {
+			var video_source = $(this).attr('data-xkit-easy-video-links-url');
+			if (video_source.indexOf("ve.media") == -1) {
+				video_source = video_source.substring(video_source.indexOf("tumblr_"));
+				if (video_source.indexOf("/") !== -1) {
+					video_source = video_source.substring(0, video_source.indexOf("/"));
+				}
+				video_source = "https://ve.media.tumblr.com/" + video_source + ".mp4";
+			}
+			var $temp = $("<input>");
+			$("body").append($temp);
+			$temp.val(video_source).select();
+			document.execCommand("copy");
```

Comment:
There should be some feedback to the user that the copy took place—maybe an `XKit.notifications.add`?

Also, is there a reason we're using the old execCommand API vs the [new async clipboard API](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText)? That would replace this entire dom manipulation hackery with a single function call